### ISSUE TITLE
How to calculate all trainable parameters??

### ISSUE THREAD

**Issue by tuandv2021** [bangoc123/mlp-mixer#3]
Thanks so much for the nice implementation.
I just have a question about parameter.

How can I calculate all trainable parameters in this model?
I have tried mlpmixer.summary(), but i do not work.

**Comment by bangoc123** [comment]
@tuandv2021 yes, it is custom model so we are unable to use summary(), you can calculate using shape of params of each layer and sum them up.

```
 self.W1 = tf.Variable(
            initial_value=w_init(shape=(S, DS), dtype="float32"),
            trainable=True,
    )
```
